R/pkg/R/mllib.R

```diff
@@ -917,14 +922,14 @@ setMethod("spark.lda", signature(data = "SparkDataFrame"),
 # Returns a summary of the AFT survival regression model produced by spark.survreg,
 # similarly to R's summary().
 
-#' @param object A fitted AFT survival regression model
+#' @param object a fitted AFT survival regression model.
 #' @return \code{summary} returns a list containing the model's coefficients,
 #' intercept and log(scale)
 #' @rdname spark.survreg
 #' @export
 #' @note summary(AFTSurvivalRegressionModel) since 2.0.0
 setMethod("summary", signature(object = "AFTSurvivalRegressionModel"),
-          function(object, ...) {
+          function(object) {
```

Comment:
I thought we need `...` in the generic for `summary` (since we have to match the base::summary signature and have variable parameters)?